javascript/photo.js

```diff
@@ -115,64 +115,65 @@ backgrounds.Photo = new Model({
         return imageData;
     },
 
+    isValidLocalStorageState: function(cacheSize) {
+        var count = localStorage.getItem("count");
+        var order = localStorage.getItem("order");
+        var cachedImages = localStorage.getItem("cachedImages");
+        return count != null && Number(count) >= 0 && order != null && JSON.parse(order).length == this.BACKUP_IMAGES_COUNT && cachedImages != null && JSON.parse(cachedImages).length <= cacheSize;
+    },
+
     /**
      * Setup LocalStorage before displaying first image.
      */
     setup: function() {
         var order = this.permutation(this.BACKUP_IMAGES_COUNT);
         localStorage.setItem("count", 0);
-        localStorage.setItem("cacheIndex", 0);
         localStorage.setItem("order", JSON.stringify(order));
         localStorage.setItem("cachedImages", JSON.stringify([]));
     },
 
     /**
      * Displays the image.
-     * TODO: Update function name & doc. This function does more than just display
+     * TODO: Update function name & doc. This function does more than just
+     * display
```

Comment:
Why was this split to 2 lines when the rest of the file exceeds this line length in many places?